lighthouse-cli/test/smokehouse/test-definitions/oopif-requests.js

```diff
@@ -47,11 +47,20 @@ const expectations = {
       'network-requests': {
         details: {
           items: {
```

Comment:
Can you add the first non-document request for each iframe as well. This would be a check to ensure we start listening for OOPIF requests in time. For me, those were:

- paulirish.com: `https://www.googletagmanager.com/gtag/js?id=G-PGXNGYWP8E`
- YT embed: `https://www.youtube.com/s/player/966d033c/www-player.css`

Reply:
gtag: ok
yt: that's gonna go stale real quick, just check for any css ?

Reply:
> that's gonna go stale real quick, just check for any css ?

Maybe look for `https://www.youtube.com/s/player/*/www-player.css`. I think looking for this specific CSS file is important. It was frequently one of the "missing" requests over in `perf-diagnostics-third-party` which also has a YT embed.

Reply:
@adamraine added.